_pages/antipatterns.md

```diff
@@ -0,0 +1,108 @@
+---
+layout: default
+title: Pair Programming Antipatterns
+permalink: /antipatterns
+meta: What you shouldn't do when pair programming.
+---
+
+{% include article-top.html %}
+
+## For navigators:
+
+### Leaping on errors too quickly
+
+Give your driver a chance to notice their own syntax errors and typos. 
+
+Constantly pointing out small errors hurts flow. Yours _and_ theirs.
+
+Remember: your job is to consider the bigger picture, not to point out misspelled words as soon as you spot them.
+
+
+### Giving low-level instructions
+
+If you have a suggestion for the driver, communicate it at the highest level of abstraction they'll understand.
+
+If you find yourself dictating code (or worse, individual keystrokes), stop and see if you can communicate your idea at a higher level. 
+
+If that fails, ask to drive for a bit to get your idea sketched out.
+
+
+### Not bringing a keyboard
+
+Bring your own keyboard to every pairing session. Plug it in before you start.
+
+This makes swapping roles easier and allows you to show rather than tell when words fail.
+
+
+## For drivers:
+
+### Driving too fast
+
+If you're highly proficient with your editor, it's easy to move fast enough to lose even experienced navigators.
+
+Unless you're sure your pair is keeping up, don't manipulate code quite as fast as you're able. 
+
+It helps if you dictate what you're doing.
+
+
+### Allowing a checked-out navigator
+
+It's easy to lose your navigator's attention by moving too fast, or doing things they don't quite understand.
+
+If you get the sense that your pair's attention is drifting, stop and sync up. 
+
+A bad question: "You understand this, right?"
+
+A good question: "Which part of this is hardest to follow?"
+
+**Pairing should involve constant two-way communication.** If you or your navigator has gone quiet, stop and check in.
+
+### Unequal screen access
```

Comment:
On my team, we often duplicate the screen on two monitors, so that each dev has the screen right in front of their eyes.

Not always practical, but very effective.

Reply:
We used to do this at all our pairing stations at previous jobs to.  Big fan of this set up

Reply:
Another note about this, if you do decide to duplicate your screen, you need to get better at mentioning the line number you are referencing or using the mouse cursor to indicate what you are talking about. When you're each looking at a different screen, pointing at yours doesn't get the job done 😂 .